document-store/src/main/java/org/hypertrace/core/documentstore/Datastore.java

```diff
@@ -1,13 +1,9 @@
 package org.hypertrace.core.documentstore;
 
-import com.typesafe.config.Config;
 import java.util.Map;
 import java.util.Set;
 
 public interface Datastore {
-
-  boolean init(Config datastoreConfig);
```

Comment:
As this was never used except internally via DataStoreProvider, we thought of removing it. 
Its not use anywhere, right?

Reply:
Yeah. Not getting used from outside clients.